README.md

```diff
@@ -107,6 +107,13 @@ Use the following `nox` session to update the dependency lock files in `tests/`.
   nox -e pip-compile
   ```
 
+To ensure that the dependency lock files are in sync with the base requirements
+files but not upgrade transitive dependencies, use the `--no-upgrade` flag:
```

Comment:
```suggestion
To synchronize dependency lock files with base requirements files without changing transitive dependencies, use the `--no-upgrade` flag:
```

Reply:
I'd try and make this less wordy. Also I'm a big advocate for one sentence per line (symantic line breaks).

Reply:
sembr.org says

> A maximum line length of 80 characters is RECOMMENDED.

Anyways, your suggestion is much clearer and more concise. Thanks!